pytom/reconstruction/reconstructionStructures.py

```diff
@@ -1491,7 +1491,7 @@ def saveParticleProjections(self, particles, projectionSize, binning=1,
 
     # ====== new general to_projection_stack functions ===================
     def to_projection_stack(self, weighting=0, binning=1, low_pass_freq=0.9, apply_circle_filter=True,
-                            scale_factor_particle=1., angle_specimen=0., particle_diameter=280, pixel_size=2.62,
+                            scale_factor_particle=1., angle_specimen=0., particle_diameter=0, pixel_size=2.62,
```

Comment:
For overwritten defaults I prefer `None`
```suggestion
                            scale_factor_particle=1., angle_specimen=0., particle_diameter=None, pixel_size=2.62,
```